Misc/NEWS.d/next/Core and Builtins/2019-04-02-20-02-22.bpo-36475.CjRps3.rst

```diff
@@ -0,0 +1 @@
+Finalize PyEval_AcquireLock() and PyEval_AcquireThread() properly
```

Comment:
I don't understand the NEWS entry. When I read the code, I understand that PyEval_AcquireLock() and PyEval_AcquireThread() are fixed to handle properly Python finalization: exit immediately the current thread if Python is finalizing.

Reply:
I have rephrased the news entry.